gnome/gstreamer1-gst-plugins-bad/Portfile

```diff
@@ -62,133 +71,87 @@ depends_lib         port:gstreamer1-gst-plugins-base \
 #
 # port:mjpegtools (mpeg2enc mplex build fails) need to disable mpeg2enc mplex explicitly in case mjpegtools is installed
 # port:faac has a restrictive license, disable faac plugin by default
 # port:libfdk-aac has a restrictive license, disable fdkaac plugin by default
-# plugin frei0r (no dependency, seqfaults on load)
 # path:lib/opencv3/libopencv_core.dylib:opencv3 (opencv plugin fails to configure with opencv 3.0.0+, can't find opencv2/bgsegm.hpp, now opencv2/video/background_segm.hpp)
 # plugin opus is now contained in gstreamer1-gst-plugins-base, disable this version
 # plugin ipcpipeline fails to build
 
 # requires support for C11 (redefinition of typedef ‘GstTestHTTPSrc’ at elements/test_http_src.c:101)
-compiler.c_standard 2011
-
-# OpenExr required C++11
+compiler.c_standard         2011
+# OpenExr requires C++11
 compiler.cxx_standard       2011
 configure.cxxflags-append   -std=gnu++11
 
-configure.args      --disable-silent-rules \
-                    --disable-fatal-warnings \
-                    --disable-dc1394 \
-                    --disable-examples \
-                    --disable-faac \
-                    --disable-fdk_aac \
-                    --disable-frei0r \
-                    --disable-ipcpipeline \
-                    --disable-lv2 \
-                    --disable-mpeg2enc \
-                    --disable-mplex \
-                    --disable-opencv \
-                    --disable-opus \
-                    --disable-srtp \
-                    --enable-static \
-                    --with-hls-crypto=nettle \
-                    --disable-apple_media
-
-# port:soundtouch (fails on autoreconf on darwin 8 & 9, see #27533) disable soundtouch on these platforms
-# plugin applemedia (Apple video) now only builds on darwin 14 and later due to use of
-# constant AVQueuedSampleBufferRenderingStatusFailed introduced in OSX 10.10
-# chromaprint 1.4+ requires libc++ (see #53072)
-# port:openal-soft build fails on 10.6 and presumably earlier
-
-platform darwin {
-    if {${os.major} < 11} {
-        depends_lib-delete      port:openal-soft
-        configure.args-append   --disable-openal
-    }
-    if {${os.major} < 10} {
-        depends_lib-delete      port:soundtouch port:x265
-        configure.args-append   --disable-soundtouch --disable-x265
-    }
-    if {${configure.cxx_stdlib} eq "libstdc++"} {
-        configure.args-append   --disable-chromaprint
-    } else {
-        depends_lib-append      port:chromaprint
-    }
-}
+configure.args-append \
+                    -Dapplemedia=disabled \
+                    -Dexamples=disabled \
+                    -Dfaac=disabled \
+                    -Dfdkaac=disabled \
+                    -Dopencv=disabled \
+                    -Dsctp=disabled \
+                    -Dvulkan=disabled
 
-configure.cppflags-append   "-L${prefix}/lib"
+configure.cppflags-append   -L${prefix}/lib
 configure.cflags-append     -std=c99 -funroll-loops -fstrict-aliasing
```

Comment:
why is this adding "c99" but then you request "2011" above?

Reply:
Hmm, it was already there like this. I guess we can change it to `-std=c11`.